test.js

```diff
@@ -0,0 +1,45 @@
+import test from 'ava';
+import kapPluginTest from 'kap-plugin-test';
+
+const response = {
+	body: '{"data":{"link":"http://imgur.com/a/Z9B4u"}}'
+};
+
+test('request and copy to clipboard', async t => {
+	const plugin = kapPluginTest('unicorn.gif');
```

Comment:
Same remark is this one https://github.com/wulkano/kap-giphy/pull/1#discussion_r119536657.

Should we add `unicorn.gif` as fixture to the repository or is it okay to just have it as dummy?